src/app/material-docs-app.scss

```diff
@@ -20,21 +16,14 @@ material-docs-app > app-component-sidenav {
   flex: 1;
 }
 
-material-docs-app > app-homepage,
-material-docs-app > app-guides,
-material-docs-app > guide-viewer {
+material-docs-app > router-outlet + * {
```

Comment:
selectors that end with `*` aren't the most efficient, is there any other way to do this?

Reply:
I don't think so as these are all going to be custom components, some that start with `app-` (like `app-guides`) and some that don't (like `guide-viewer`. Hopefully the `+` limits the scope of the matching for `*` enough that this isn't too inefficient.

Reply:
Selectors are parsed right-to-left, so the `+` won't help. Is there some way we could apply a common class to all of them instead?

Reply:
Done